sklearn/feature_extraction/tests/test_text.py

```diff
@@ -484,6 +485,17 @@ def test_tfidf_vectorizer_setters():
     assert_true(tv._tfidf.sublinear_tf)
 
 
+def test_tfidf_sort_sets_when_pickling():
+    vocab_words = ['beer', 'burger', 'celeri', 'coke', 'pizza',
+                   'salad', 'sparkling', 'tomato', 'water']
+    for x in xrange(0, 100):
+        vocab_set = set(random.sample(vocab_words, 5))
+        tv = TfidfVectorizer(vocabulary=vocab_set)
+        unpickled_tv = pickle.loads(pickle.dumps(tv.fit(ALL_FOOD_DOCS)))
+        unpickled_tv.fit(ALL_FOOD_DOCS)
+        assert_equal(tv.get_feature_names(), unpickled_tv.get_feature_names())
```

Comment:
Since TfidfVcctorizer inherits from CountVectorizer, I thought it'd suffice to test CountVectorizer.